### PR TITLE
[ADD] accounting: inter bank transfers

### PR DESCRIPTION
**Pull request by tivisse**
https://docs.google.com/document/d/1d7vLhTrfeRySvab6IQ1y_1X4z6eiJDy5D7nXkxnIR0o/edit

### CODE REVIEW

File: accounting/bank/misc/interbank.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+------------------------------------
+
+Note that the bank balance computed by Odoo is different that the last
+statement of your bank.
+
+.. image:: media/interbank11.png
+   :align: center
+
+That is because we did not import the bank statement confirming the
+departure and arrival of the money. It’s thus necessary to import your
```
Comment (tivisse) [inline accounting/bank/misc/interbank.rst]: Beurk

File: accounting/bank/misc/interbank.rst [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+   :align: center
+
+Create a second bank account / Journal
+--------------------------------------
+
+Before we can register an internal transfer we need to add a new bank to
+our accounting dashboard. To do so enter the accounting module, click on
+:menuselection:`Configuration --> Bank Accounts`. Create a new bank account. You should
+fill in the **Account Number**. You can also create and edit your bank to
+specify your bank’s details.
```
Comment (tivisse) [inline accounting/bank/misc/interbank.rst]: beurk ’

File: accounting/bank/misc/interbank.rst [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+.. image:: media/interbank01.png
+   :align: center
+
+Create a new payment. The payment type will automatically be set to
+internal transfer. Select the **Bank** you want to transfer to, specify the
+**Amount** and add a **Memo** if you wish.
+
+.. note::
+    The memo is important if you wish to automatically reconcile (`see last chapter <Reconcile_>`_).
```
Comment (mart-e) [inline accounting/bank/misc/interbank.rst]: "last chapter" for something later in the doc?

File: accounting/bank/misc/interbank.rst [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+That is because we did not import the bank statement confirming the
+departure and arrival of the money. It's thus necessary to import your
+bank statement and reconcile the payment with the correct bank statement
+line. Once you receive your bank statements click the **new statement**
+button of the corresponding bank to import them.
+
+.. image:: media/interbank07.png
+   :align: center
+
+Fill in your **Transactions line**. Ones done, Odoo will display a **Computed
```
Comment (mart-e) [inline accounting/bank/misc/interbank.rst]: "Ones done" -> "Once done"

File: accounting/bank/misc/interbank.rst [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+You need to choose counterparts for the paiement. Select the correct
+bank statement line corresponding to the paiement and click on the 
+**reconcile** button. Close the statement to finish the transaction
+
+.. image:: media/interbank08.png
+   :align: center
+
+The same steps will need to be repeated once you receive your second
+bank statement. Note that if you specify the correct amount, and the
+same memo in both bank statement and payment transaction the
```
Comment (mart-e) [inline accounting/bank/misc/interbank.rst]: don't get that, probably missing a word or punctuation

**Pull request merged**